de.tu_bs.cs.isf.e4cf.compare.data_structures/src/de/tu_bs/cs/isf/e4cf/compare/data_structures/io/reader/VisitorUtil.java

```diff
@@ -0,0 +1,55 @@
+package de.tu_bs.cs.isf.e4cf.compare.data_structures.io.reader;
+
+import de.tu_bs.cs.isf.e4cf.compare.data_structures.impl.NodeImpl;
+import de.tu_bs.cs.isf.e4cf.compare.data_structures.interfaces.Node;
+
+/**
+ * This class provides default methods for @see Visitor to remove redundancy and enforce standards.
+ * 
+ * @author Paulo Haas
+ *
+ */
+
+public class VisitorUtil {
+	/**
+	 * Creates a leaf node
+	 * 
+	 * @param n JavaParser Node
+	 * @param arg Parent Node
+	 */
```

Comment:
Starting method names with a capital letter is not a Java Conevention. I think its a c# convention.